docs/python/python-tutorial.md

```diff
@@ -138,7 +138,7 @@ Finally, save the file (`kb(workbench.action.files.save)`). At this point, you'r
 
 For full details on editing, formatting, and refactoring, see [Editing code](/docs/python/editing.md). The Python extension also has full support for [Linting](/docs/python/linting.md).
 
-## Run Hello World
+## Run Python Code
```

Comment:
With this H2 header changed, you'll need to update the docs/languages/python topic which has a direct link to the header.

Search for this and I think it's fine to just link to the topic rather than the header:
```
For a more specific walkthrough and other ways of running code, see the [run code tutorial](/docs/python/python-tutorial.md#run-hello-world).
```

becomes

```
For a more specific walkthrough and other ways of running code, see the [run code tutorial](/docs/python/python-tutorial.md).
```

Reply:
Ah, nice catch! Pushed those changes :)

Reply:
Another nit: we like to use sentence casing for headers so "Run Python code"
Will also prevent possible confusion with "VS Code" (uppercase)